README.md

```diff
@@ -17,3 +17,78 @@ Build platform for listing software engineers using Next.js, Typescript, Prisma,
 - Should be possible to create new dev profile
 - Should be possible to edit dev profile
 - Should be possible to authorize with GitHub
+
+### Project Documentation
+
+**Branch Management and Merge:**
+
+-Each task should have its own branch in the Git repository. The branch name should be descriptive and reflect the task's topic.  
+-Before merging into the main branch (main), at least three (3) approvals from other team members are required.
+
+**Branch and Commit Naming:**
+
+We follow the convention specified in: [Link](https://www.conventionalcommits.org/en/v1.0.0/")
+For example: 'feat/branch-name', 'fix: commit name'
+
+**Communication and Code Review:**
+
+-We actively monitor the Discord channel, where all pull requests (PRs) are automatically posted.  
```

Comment:
**nitpick:** Add spacing because it's not formatting properly when `-` is sticking to `We`